### PR TITLE
Export the gmf-search input value

### PR DESCRIPTION
**Pull request by fredj**
To be able to use the value outside of the directive

### CODE REVIEW

File: contribs/gmf/src/directives/search.js [hunk 1]
```diff
@@ -117,6 +118,9 @@ gmf.module.value('gmfSearchTemplateUrl',
 gmf.searchDirective = function(gmfSearchTemplateUrl) {
   return {
     restrict: 'E',
+    bindToController: {
+      'input_value': '=?gmfSearchInputValue'
```
Comment (tsauerwein) [inline contribs/gmf/src/directives/search.js]: `inputValue`? (without underscore)

Comment (fredj) [inline contribs/gmf/src/directives/search.js]: it's the variable name that was already in gmf-search (but I can change it)

Comment (fredj) [inline contribs/gmf/src/directives/search.js]: renamed to inputValue

**Comment by tsauerwein** [comment]
Thanks! :+1:

**Comment by fredj** [comment]
Thanks for the review

**Pull request merged**